### DISCUSSION TITLE
Probleme bei WR Regelung Hoymiles HMS-1000-2T

### DISCUSSION THREAD

**Discussion by CSC01de** [hoylabs/OpenDTU-OnBattery#1455] [Q&A] [answered]
Hallo,

ich habe mir ebenfalls eine OpenDTU on Battery eingerichtet und die Regelung (Nulleinspeisung) funktioniert am Anfang sehr gut.
Leider stellt der WR nach ca einer Woche die Produktion ein.,bzw. drosselt die Leistung auf 1 - 3 W runter.
Alle Spannungen an den Eingängen sind ausreichend für eine Produktion. Im Fehlerspeicher der DTU steht der Fehler 223 an. Gibt aber wohl keine genaue Beschreibung.  
Wenn ich in der DTU einen WR Neustart auslöse fährt der WR die vorgegebene Leistung an.
Hat einer von euch auch so ein Problem?

**Comment by spcqike** [comment, marked as answer]
Hast du einen nächtlichen Neustart des wechselrichters eingestellt?

**Reply by CSC01de** [reply]
Nein, habe keinen Neustart des Wechselrichters an der DTU eingestellt. Muss ich das in geringen Abständen machen? Danke schonmal für deine Hilfe.

**Reply by spcqike** [reply]
Naja. Täglich wäre gut. Dafür gibt es die Funktion. Du kannst eh nur eine Uhrzeit einstellen, kein Intervall :)

**Reply by Gururom** [reply]
Da ich seit neusten einen Akku dran habe, läuft meiner nun auch durch, wo finde ich die Funktion, habe gerade danach gesucht und nichts gefunden.
Nutze 22.10.24 aber einen Bluepalm bzw 2 davon zwischen Modulen und WR.

**Reply by CSC01de** [reply]
Das steht unter Dynamic Power Limiter Einstellungen.  Wechselrichtereinstellungen .
Uhrzeit für geplanten NEUSTART

**Reply by spcqike** [reply]
die Funktion geht nur, wenn der Wechselrichter nicht "von Solarmodulen" betrieben wird. 
der Bluepalm  wird jedoch wahrscheinlich Solarmodule "simulieren", oder? Da musst du dir leider etwas anderes einfallen lassen, wie du den ab und an neustarten kannst.

**Reply by Gururom** [reply]
> die Funktion geht nur, wenn der Wechselrichter nicht "von Solarmodulen" betrieben wird. der Bluepalm wird jedoch wahrscheinlich Solarmodule "simulieren", oder? Da musst du dir leider etwas anderes einfallen lassen, wie du den ab und an neustarten kannst.

Genau so ist es, muss ich dann über einen Shelly in der Zuleitung regeln.
Trotzdem Danke..

**Reply by spcqike** [reply]
> muss ich dann über einen Shelly in der Zuleitung regeln

ich bin mir nicht sicher, ob der Wechselrichter neustartet, wenn man die AC Leitung trennt. Immerhin speist er sich aus der DC-Seite. und ein regelmäßiges Trennen der DC-Seite ist auf Dauer auch nicht zu empfehlen.

besser wäre ein extern laufendes Script, welches den Neustartbefehl per web API schickt. falls das geht. Müsste aber eigentlich, irgendwie.
https://opendtu-onbattery.net/firmware/web_api/

**Reply by Gururom** [reply]
Mit solchen Scripten etc hab ich ja nun überhaupt keine Ahnung.
Was müsste mann denn da genau und eben auch wo in der opendtuoB Software einfügen um einen Neustart zu initiieren?

**Reply by spcqike** [reply]
In openDTU nicht. 
Du bräuchtest ein externes Gerät, welches ein Web Request an die openDTU schicken kann. Ein raspberry, ein anderweitig vorhandener Server mit Smart Home Software wie ioBroker, Home Assistant, FHEM, node-red,….. 
ja, vielleicht ginge es sogar mit einem tasmota gerät oder einem Shelly, aber da bin ich mir nicht ganz sicher. 

Oder, du machst ein Feature Request auf und beschreibst dein Problem und ne mögliche Lösung. Immerhin kann openDTU onBattery es ja prinzipiell. Nur eben ist es für „solar only“ Systeme aktuell nicht eingebaut, da Systeme, die tatsächlich an solar Modulen hängen, nachts so oder so aus sind und daher Neustarten. 
Mit dem Aufkommen dieser, ich sag mal, einfachen „all in one batteriespeicher“, die nach außen ein solarmodul simulieren, verschiebt sich das ja immer mehr.

**Reply by schlimmchen** [reply]
> muss ich dann über einen Shelly in der Zuleitung regeln.

Auf keinen Fall versuchen die DC Leitung des WR mit einem Shelly zu schalten! AC schalten reicht nicht.

@Gururom Warum ist es wichtig, dass der WR als "wird an Solarmodulen betrieben" gekennzeichnet ist? Er hängt doch an einer Batterie. Hoffentlich mit allen Eingängen gleichermaßen?

**Reply by Gururom** [reply]
ja, 2xBluepalm mit je 2 Ausgängen, also eigentlich alles gleichmäßig
ok, ich kann das ja mal rausnehmen, nur was dann alles einzustellen wäre.....
ich meine ja nur, da die Akkus nicht von openDTU erreichbar sind.
